bitnami/external-dns/values.yaml

```
@@ -14,6 +15,10 @@ global:
   ##   - myRegistryKeySecretName
   ##
   imagePullSecrets: []
+    
+  commonLabels: {}
+  ## e.g:
+  ## team_name: dev
```

In order to follow the same approach used by the rest of parameters:
```suggestion
  ## E.g.
  ## commonLabels:
  ##   team_name: dev
  ##
  commonLabels: {}
```